File: 백준/Bronze/20674. Statistics/Statistics.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int tmp,prev=1000,rlt=0,n = scan.nextInt();
        for(int i=0;i<n;i++) {
            tmp=scan.nextInt();
            if(prev>=tmp)
                prev=tmp;
            else
                rlt+=tmp-prev;
        }
        System.out.print(rlt);
    }
}

File: 백준/Bronze/20674. Statistics/Statistics.c
#include <stdio.h>
int main(void) {
    int n,prev=1000,rlt=0,tmp;
    scanf("%d",&n);
    for(int i=0;i<n;i++) {
        scanf("%d",&tmp);
        if(prev>=tmp)
            prev=tmp;
        else
            rlt+=tmp-prev;
    }
    printf("%d",rlt);
    return 0;
}